build/build.sh

```diff
@@ -35,7 +35,7 @@ fi
 LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildStatus=${GITSTATUS}"
 
 : ${GITTAG:=$(git describe 2> /dev/null || echo "unknown")}
-LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildTag=${GITTAG}"
+LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildTag=${GITTAG} -X ${REPO_PATH}/pkg/version.minimumSupportedVersion=${MINIMUM_SUPPORTED_VERSION}"
```

Comment:
Where does `$MINIMUM_SUPPORTED_VERSION` come from?

Reply:
From the spec file, on the build line.  If not present it defaults to "" which is default for all versions. 

See https://github.com/maistra/istio-operator/pull/672

Reply:
spec file for product will now contain

-COMMUNITY=%{community_build} GO111MODULE=on VERSION=%{version}-%{release} GITREVISION=%{git_shortcommit} GITSTATUS=Clean GITTAG=%{version} GIT_UPSTREAM_REMOTE="_DUMMY_" OFFLINE_BUILD="true" make compile collect-resources
+COMMUNITY=%{community_build} GO111MODULE=on VERSION=%{version}-%{release} GITREVISION=%{git_shortcommit} GITSTATUS=Clean GITTAG=%{version} GIT_UPSTREAM_REMOTE="_DUMMY_" OFFLINE_BUILD="true" MINIMUM_SUPPORTED_VERSION=${MINIMUM_SUPPORTED_VERSION} make compile collect-resources